plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs

```diff
@@ -69,7 +69,7 @@ How do we know whether we're using a haskell.nix GHC? Well, we can't in general,
 an extra-special patched version which we *can* detect, so that lets us do the job for now until we can
 fix this properly.
 -}
-#if __GLASGOW_HASKELL_PATCHLEVEL2__ == 20210212
+#if (__GLASGOW_HASKELL_PATCHLEVEL2__ == 20210212) || defined(UBXT_PATCH)
```

Comment:
might as well just bin the first check and set the module option right now?

Note that all of our downstream folks will need to do this too.

Reply:
Yes, I've don this for @sevanspowell in plutus#4325. I'll see if we can change the UBXT patch to be less invasive tomorrow.

Reply:
I think I'm missing something. I'm just asking about setting the module option in this PR and getting rid of the `__GLASGOW_HASKELL_PATCHLEVEL` conditional now.